### ISSUE TITLE
null sequence in TUPLET_THREE

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#159]
[Sentimental Journey (2).zip](https://github.com/Audiveris/audiveris/files/1999550/Sentimental.Journey.2.zip)

Measure 3 seems to make problems (no rhythm lines in output view). And I get the following error message when reprocessing rhythm:
TupletInter{#14552 MANUAL(1.000) s:2 TUPLET_THREE} null sequence in TUPLET_THREE: 0
Error java.lang.IllegalArgumentException: no such vertex in graph: TupletInter{#13673 REMOVED(0.523/0.523) s:2 TUPLET_THREE} checkConfig

**Comment by hbitteur** [comment]
The "TupletInter{#14552 MANUAL(1.000) s:2 TUPLET_THREE} null sequence in TUPLET_THREE: 0" should be jsut a debug message, not an info message. This is fixed.

But there is a problem in this measure: there are beam hooks that have been manually added (and their beam group is not set, leading to an exception when measure is reloaded). This could explain why the measure was seen as empty.
I need further investigation.

**Comment by hbitteur** [comment]
There were messy bugs related to order in which additions and links were made (not proud of this, sorry).
Fixed in commit 9b4361b624556e79f51373dbe81891c39f3dee8c

I played a lot with this really poor quality score. It needed a lot of manual actions (creation of measures, stems, heads, beams, links to be set, ...)
Since this had an impact on voice assignment, I tried a modification in score display: Colored voices are now available in physical mode (they were already available in logical mode). 
I found that this is a good help for manual edition, since you can see immediately if voices evolve as you would expect from your manual actions. But, try it by yourself and give me your feedback.

Caveat: Voices "should" be updated after each user action, but I must I have forgotten a few triggering cases, more work is needed with links. In case of doubt, you can force update with measure reprocess rhythm.

Use the standard toggle switch on tool bar to display or not voices colors.
Done in commit da25d167f590b0c3b1faeeb97feb80ec85396598

**Comment by Bacchushlg** [comment]
Great improvement!
Due to the colouring, I now can see immediately if something is wrong with the assignment. 
I was able to "repair" almost all errors - mostly by removing the "wrong" note head and re-add it, giving the algorithm the chance to re-assign correctly.
Especially there were some cases where 2 stems in line were not detected as a single one. But I simply removed one of them an assigned the note head to the other.
